frontend/app/work_packages/controllers/menus/work-package-context-menu-controller.js

```diff
@@ -99,6 +99,11 @@ module.exports = function(
     return WorkPackagesTableHelper.getWorkPackagesFromRows(selectedRows);
   }
 
+  function getSelectedWorkPackageId() {
+    var selected = getSelectedWorkPackages()[0];
+    return selected && selected.id;
+  }
+
```

Comment:
You removed this in https://github.com/opf/openproject/pull/3835/files#diff-ab5294bb05227591715f26738a3b98aeL90